docs/DeveloperGuide.md

```diff
@@ -805,23 +805,55 @@ testers are expected to do more *exploratory* testing.
     1. Prerequisites: List all students using the `list` command. Multiple students in the list.
 
     1. Test case: `delete 1`<br>
-       Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message.
-       Timestamp in the status bar is updated.
+       Expected: First st is deleted from the list. Details of the deleted student shown in the result display box.
 
     1. Test case: `delete 0`<br>
-       Expected: No student is deleted. Error details shown in the status message. Status bar remains the same.
+       Expected: No student is deleted. Error details shown in the result display box.
 
-    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+    1. Other incorrect delete commands to try: `delete`, `delete x`,  (where x is larger than the list size)<br>
```

Comment:
1. Other incorrect delete commands to try: `delete`, `delete x`, (where x is larger than the list size)<be>

- extra white space before the bracket